### DISCUSSION TITLE
8 Days to Bun on Windows

### DISCUSSION THREAD

**Discussion by seneca** [nuejs/nue#180] [Q&A] [answered]
I hope that Bun on windows will alleviate some of the problems the Windows users experience when running Nue. :-)
https://twitter.com/bunjavascript/status/1749828267190911297

**Comment by tipiirai** [comment]
Tomorrow that is. Need to buy a Windows machine after Nue 1.0 is out and things get more serious

**Reply by nobkd** [reply]
Windows support will take them a few more days:

> Bun for Windows will ship on Friday afternoon instead of Wednesday. Needs a couple more days to bake.

https://twitter.com/bunjavascript/status/1752348374190506077?s=19

https://github.com/oven-sh/bun/issues/43#issuecomment-1918350509

**Reply by nobkd** [reply]
https://twitter.com/bunjavascript/status/1753267105909416079

> [!NOTE]
> We are delaying Bun for Windows until February 15 to get the test suite to pass 100%  
>
> To help report bugs, run this:
>
> `iwr bun.sh/install.ps1 | iex`

**Reply by nobkd** [reply]
https://bun.sh/blog/bun-v1.0.28#windows-is-not-ready-yet

> ## Windows is not ready yet
>
> We are still working on Windows. There's a big PR we aim to merge on Wednesday, which will get us closer.

**Reply by nobkd** [reply]
https://bun.sh/blog/bun-v1.0.31#windows-support-is-close

> We are close to shipping Windows support with Bun v1.1. Once Bun for Windows passes 95% of Bun's test suite, we will announce the release date.
>
> Bun for Windows currently passes 92.51% of Bun's test suite

**Reply by nobkd** [reply]
https://bun.sh/blog/bun-v1.0.36#bundows-bun-v1-1-is-april-1st

> Bundows (Bun v1.1) is April 1st
> 
> Bun v1.1 will ship on April 1st.

(Sounds almost like an April's fools joke)

**Reply by nobkd** [reply, marked as answer]
https://youtu.be/yXTFOeGly9o
https://bun.sh/blog/bun-v1.1

Bun for Windows is here!

**Comment by fritx** [comment]
Parallel Desktop can also be an option to run Windows in MacOS xD, I tried its free trial 2 years ago.
